docs/data/charts/pyramid-demo/pyramid.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Charts - Pyramid demonstration
+productId: x-charts
+components: FunnelChart, FunnelPlot
+---
+
+# Charts - Pyramid demonstration
+
+<p class="description">This page demonstrates the pyramid chart.</p>
+
+## Pyramid Chart
+
+The pyramid chart is a variation of the funnel chart.
+
+To create a pyramid chart, set the `curve` property to `pyramid` in the series.
+
+{{"demo": "Pyramid.js"}}
```

Comment:
I would make this example a pyramid by using a different dataset with data from `small > big`

Reply:
First and third (playground) examples are now pyramids. 

I tried reversing the order of the series in `populationByEducationLevel.ts` file, but couldn't. Can you take a look? Currently, it just shows the data without value formatter or labels.